frontend/src/pages/connectionTypes/columns.ts

```diff
@@ -0,0 +1,78 @@
+import { SortableData } from '~/components/table';
+import { ConnectionTypeConfigMapObj } from '~/concepts/connectionTypes/types';
+import { ownedByDSC } from '~/concepts/k8s/utils';
+import { PreInstalledName } from '~/pages/connectionTypes/const';
+
+const sorter = (
+  a: ConnectionTypeConfigMapObj,
+  b: ConnectionTypeConfigMapObj,
+  keyField: string,
+): number => {
+  let compValue = 0;
+
+  if (keyField === 'creator') {
+    const aValue = ownedByDSC(a)
+      ? PreInstalledName
+      : a.metadata.annotations?.['opendatahub.io/username'] || 'unknown';
```

Comment:
Create a util to get the display username as this logic is duplicated elsewhere.